test/shmemx/Makefile.am

```diff
@@ -90,10 +89,14 @@ LDADD += $(top_builddir)/pmi-simple/libpmi_simple.la
 endif
 
 if SHMEMX_TESTS
+
+if HAVE_PTHREADS
```

Comment:
The `if` here should not be necessary.  More ifs == more opportunities for automake to automake you sad.

Reply:
This `if` is not new to this PR, but that's reasonable.  Want to remove it?